File: Lista_5/lista5_ex6.c
/* Exercício 6:
Escreva uma função que recebe um vetor de números reais e o seu tamanho por parâmetro e devolve a média aritmética dos números do vetor. 
O protótipo da função deve ser:
double media(double v[], int tam);
*/
#include <stdio.h>

double media(double v[], int tam);

int main() {
int tam, i;
double v[1000];

printf("Quantos numero tera o vetor: ");
scanf("%d", &tam);

for(i=0; i < tam; i++) {
    printf("\nDigite o numero %d: ", i+1);
    scanf("%lf", &v[i]);
}

printf("\nA media aritmetica e: %.1lf\n", media(v, tam));

return 0;
}

double media(double v[], int tam) {
    double sum=0;

    for(int i=0; i < tam; i++) {
        sum = sum + v[i];
    }
return sum/tam;
}

File: Lista_5/Lista5_ex6.py
def media(v, tam):
	sum = 0
	i = 0
	for i in range(tam):
		sum = sum + v[i]
	return sum / tam	

def main():
	i = 0
	v = []
	tam = int(input("O tamanho do vetor: "))

	print("\nInsira os valor no vetor:")
	for i in range(tam):
		r = float(input())
		v.insert(i, r)

	print("\nA media aritmetica e: %.2f\n" % media(v, tam))

if __name__ == "__main__":
	main()
